packages/components/src/components/select/select-option/select-option-styles.ts

```diff
@@ -0,0 +1,111 @@
+import type { Theme } from '../../../types';
+import { getCss, isHighContrastMode } from '../../../utils';
+import { getNoResultsOptionJssStyle } from '../../../styles/option-styles';
+import {
+  addImportantToEachRule,
+  getHighContrastColors,
+  getThemedColors,
+  getTransition,
+  hostHiddenStyles,
+  hoverMediaQuery,
+  prefersColorSchemeDarkMediaQuery,
+} from '../../../styles';
+import { borderRadiusSmall, fontLineHeight, spacingStaticSmall } from '@porsche-design-system/utilities-v2';
+import { Styles } from 'jss';
+
+export const getComponentCss = (theme: Theme): string => {
+  return getCss({
+    '@global': {
+      ':host': addImportantToEachRule({
```

Comment:
This has no `display` style so it is probably `inline` by default. Not sure if this is fine.
In case there are no other styles, it looks like everything is important and therefore both `:host` and `::slotted(img)` could be combined in a single object and passed to `addImportantToEachRule` instead of calling it twice with object composition.